utils/onSubmit.js

```diff
@@ -0,0 +1,21 @@
+export default function onSubmit(event) {
+  event.preventDefault();
+  const submittedForm = {
+    id: nanoid(),
+    experiences: [
+      { anger: event.target.anger.checked },
+      { enjoyment: event.target.enjoyment.checked },
+      { fear: event.target.fear.checked },
+      { disgust: event.target.disgust.checked },
+      { sadness: event.target.sadness.checked },
+    ],
+    text: event.target.text.value,
+    slider: rangeValue,
+
+    date: new Date().toLocaleString(),
+  };
+
+  setAllEntries([...allEntries, submittedForm]);
+  console.log([...allEntries, submittedForm]);
+  event.target.reset();
+}
```

Comment:
You can delete this. 
You have already implemented this code in the _app.js, so I don't think you are using it anywhere